packages/terra-modal/src/ModalContent.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { PropTypes } from 'react';
+import FocusTrap from 'focus-trap-react';
+import ModalOverlay from './ModalOverlay';
+
+const propTypes = {
+  ariaLabel: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
+  classNameModal: PropTypes.string,
+  classNameOverlay: PropTypes.string,
+  closeOnOutsideClick: PropTypes.bool,
+  closePortal: PropTypes.func,
+  role: PropTypes.string,
+};
+
+const defaultProps = {
+  ariaLabel: null,
+  children: null,
+  classNameModal: null,
+  classNameOverlay: null,
+  closeOnOutsideClick: true,
+  role: 'dialog',
+};
+
+
+class ModalContent extends React.Component {
+
+  componentDidMount() {
+    // Disable background scrolling
+    document.body.style.overflow = 'hidden';
+  }
+
+  componentWillUnmount() {
+    // Enable background scrolling
+    document.body.style.overflow = null;
```

Comment:
This could potentially cause some thrashing if the consuming application has styles set on the body